docs/dense-retrieval.md

```diff
@@ -166,6 +166,31 @@ To evaluate with on-the-fly query encoding with our pretrained encoder model
 on [Hugging Face](https://huggingface.co/castorini/tct_colbert-msmarco/tree/main) add
 `--encoder castorini/tct_colbert-msmarco`. The encoding will run on CPU by default. To enable GPU, add `--device cuda:0`.
 
+## TREC2019-DL Document Ranking (Zero Shot)
+TREC-2019 DL document ranking task, dense retrieval with TCT-ColBERT trained on MS MARCO passages, brute force index.
+
+```bash
+$ python -m pyserini.dsearch --topics dl19_doc \
+                             --index msmarco-doc-tct_colbert-bf \
+                             --encoder castorini/tct_colbert-msmarco \
+                             --output run.dl19.tct_colbert.txt \
+                             --hits 1000 \
+                             --max-passage \
+                             --max-passage-hits 100 \
+                             --msmarco \
+                             --batch 72 \
+                             --threads 72
+```
+
+We can use the official TREC evaluation tool `trec_eval` to compute metrics such as recall and ndcg
+For that we first need to convert runs and qrels files to the TREC format:
+```bash
+$ python tools/scripts/msmarco/convert_msmarco_to_trec_run.py --input runs/run.dl19.tct_colbert.txt --output runs/run.dl19.tct_colbert.trec
+$ tools/eval/trec_eval.9.0.4/trec_eval -c -mrecall.100 -mndcg_cut.3 tools/topics-and-qrels/qrels.dl19-doc.txt runs/run.dl19.tct_colbert.trec
+recall_100            	all	0.3415
+ndcg_cut_3            	all	0.6800
```

Comment:
I think ndcg@10 is the official metric? look in the overview paper to verify? report that?

Reply:
will change to msmsrco and report the corresponding metrics